en/yohane.md

```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
 - `!fokabot reconnect` - Reconnect YohaneBOT if he's not on online users list anymore  
```

Comment:
Yoooo BOT nya laki? he???

Reply:
change to `!bot reconnect` also `"she"` because that is from ripple bot

Reply:
I will also take the opportunity to correct the sentence